src/libs/ReportActionsUtils.js

```diff
@@ -97,13 +97,14 @@ function hasCommentThread(reportAction) {
  * Returns the parentReportAction if the given report is a thread/task.
  *
  * @param {Object} report
+ * @param {Object} [allReportActionsParam]
  * @returns {Object}
  */
-function getParentReportAction(report) {
+function getParentReportAction(report, allReportActionsParam) {
```

Comment:
If its optional should we give it an empty object as a default value?
```suggestion
function getParentReportAction(report, allReportActionsParam = {}) {
```